src/actions.js

```diff
@@ -137,16 +137,20 @@ export const view = () => ({
   mode: "view"
 });
 
-export const unfocus = (state: Types.StoreState<*>) => {
+export const backspace = (state: Types.StoreState<*>) => {
```

Comment:
This action should probably be named `clear()` or similar. The action names describe what they do not who calls them.